src/components/controls/TextArea/TextArea.scss

```diff
@@ -9,6 +10,10 @@ $block: '.#{variables.$ns-new}text-area';
     width: 100%;
     position: relative;
 
+    &_mobile {
+        font-size: control-variables.$mobile-safari-font-size;
```

Comment:
Not sure if it's a good idea to hardcode 16px for all the sizes in mobile devices

Reply:
It's really annoying bug. Why not?